plugins/rikki/heroeslounge/classes/matchmaking/Swiss.php

```diff
@@ -21,6 +21,12 @@ public function createMatches($pairings, $div) {
         $timeoffset = ($pairings[0][0]->region_id - 1)*36000;
         $schedule_date = date('Y-m-d H:i:s', strtotime('next sunday 21:55')+$timeoffset);
         $tbp = date('Y-m-d H:i:s', strtotime('+1 weeks sunday 21:55')+$timeoffset);
+        
+        // if it's the last round, games must be played until next friday (+5 days)
+        if ($div->season->current_round == 10) {
+	        $tbp = date('Y-m-d H:i:s', strtotime('+5 days sunday 21:55')+$timeoffset);
```

Comment:
'+1 weeks friday' is correct here - I think +5 days sunday would be equivalent to next sunday (for code that runs on Monday), i.e. ~7 days instead of ~12 days

Reply:
I'm not familiar with php, but I just tested it in PHP sandbox. Both `+5 days sunday` and `+1 weeks friday` have the same result (in this case Friday, 30.07). However, I think that `+1 weeks friday` is easier to understand and more consistent